src/api/app/views/webui2/webui/groups/_group_members.html.haml

```diff
@@ -0,0 +1,45 @@
+- write_access = policy(group).update?
+
+- if write_access
+  = link_to('#', class: 'nav-link', data: { toggle: 'modal', target: '#add-group-user-modal' }) do
+    %i.fas.fa-plus-circle.text-success
+    Add User
```

Comment:
This currently expands over the full screen. I haven't had time to debug this yet. If someone knows a fix for this, please let me know.